engine/src/test/java/com/redhat/cloud/notifications/templates/common/TestAdvisorTemplate.java

```diff
@@ -159,6 +159,7 @@ private void checkNewRecommendationsBodyResults(Action action, final String resu
 
     @Test
     public void testInstantEmailTitleForDeactivatedRecommendation() {
+        eventTypeDisplayName = "Deactivated recommendation";
```

Comment:
This looks unused.

Reply:
Used by parent class [EmailTemplatesRendererHelper](https://github.com/RedHatInsights/notifications-backend/blob/master/engine/src/test/java/com/redhat/cloud/notifications/EmailTemplatesRendererHelper.java#L149).